src/components/interfaces/MOBILE_API.xml

```diff
@@ -5369,65 +5454,149 @@
 
   <function name="OnEncodedSyncPData" functionID="OnEncodedSyncPDataID" messagetype="notification" >
     <description>
-    	Callback including encoded data of any SyncP packets that SYNC needs to send back to the mobile device.
-    	Legacy / v1 Protocol implementation; responds to EncodedSyncPData.
-    	*** DEPRECATED ***
+      Callback including encoded data of any SyncP packets that SYNC needs to send back to the mobile device.

```

Comment:
@AKalinich-Luxoft please avoid fixing formating. It is not clear what changes actually done